neo/rawio/spikegadgetsrawio.py

```diff
@@ -75,10 +81,31 @@ def __init__(self, filename="", selected_streams=None):
         BaseRawIO.__init__(self)
         self.filename = filename
         self.selected_streams = selected_streams
+        self.interpolate_dropped_packets = interpolate_dropped_packets
 
     def _source_name(self):
         return self.filename
 
+    def _produce_ephys_channel_ids(self, n_total_channels, n_channels_per_chip):
+        """Compute the channel ID labels
+        The ephys channels in the .rec file are stored in the following order:
+        hwChan ID of channel 0 of first chip, hwChan ID of channel 0 of second chip, ..., hwChan ID of channel 0 of Nth chip,
+        hwChan ID of channel 1 of first chip, hwChan ID of channel 1 of second chip, ..., hwChan ID of channel 1 of Nth chip,
+        ...
+        So if there are 32 channels per chip and 128 channels (4 chips), then the channel IDs are:
+        0, 32, 64, 96, 1, 33, 65, 97, ..., 128
+        See also: https://github.com/NeuralEnsemble/python-neo/issues/1215
+        """
+        x = []
+        for k in range(n_channels_per_chip):
+            x.append(
+                [
+                    k + i * n_channels_per_chip
+                    for i in range(int(n_total_channels / n_channels_per_chip))
+                ]
+            )
+        return [item for sublist in x for item in sublist]
+
```

Comment:
This is now in main. Let us know if you want to keep this open and try to get the digital channels working with Neo or whether you want to close this PR now?

Reply:
It's up to @rly, but I think it may be best to just keep this open and work towards getting the analog and digital IOs loadable with Neo.